keps/0035-json-schema-export.md

```diff
@@ -0,0 +1,265 @@
+---
+kep-number: 35
+short-desc: JSON-schema export
+title: JSON-schema export
+authors:
+  - "@aneumann82"
+owners:
+  - "@aneumann82"
+creation-date: 2020-09-22
+last-updated: 2020-09-29
+status: provisional
+see-also:
+  - KEP-33
+---
+
+# JSON-schema export
+
+## Table of Contents
+
+<!--ts-->
+   * [JSON-schema export](#json-schema-export)
+      * [Table of Contents](#table-of-contents)
+      * [Summary](#summary)
+      * [Motivation](#motivation)
+         * [Goals](#goals)
+         * [Non-Goals](#non-goals)
+      * [Proposal](#proposal)
+         * [User Stories](#user-stories)
+            * [Story 1](#story-1)
+         * [Implementation Details](#implementation-details)
+            * [Additional Types:](#additional-types)
+            * [Additional Attributes](#additional-attributes)
+            * [Groups](#groups)
+         * [JSON-Schema export](#json-schema-export-1)
+      * [Alternatives](#alternatives)
+      * [Implementation History](#implementation-history)
+
+<!-- Added by: aneumann, at: Tue Sep 29 15:37:14 CEST 2020 -->
+
+<!--te-->
+
+## Summary
+
+Extend the existing `params.yaml` with metadata and new types and add an export as JSON-schema.
+
+## Motivation
+
+JSON-schema is the preferred way to generate a user-friendly UI to configure a new instance of a KUDO operator or to trigger plans with parameters. KUDO should be able to export a JSON-schema that describes the required parameters.
```

Comment:
preferred way to trigger plans with parameters? 🤔 what does that mean?

Reply:
I have no idea... Probably left over from somewhere? Fixed it, thanks.